plugins/plugin-webpack/plugin.js

```diff
@@ -161,7 +172,20 @@ module.exports = function plugin(config, args) {
   if (!cssOutputPattern.endsWith(".css")) {
     throw new Error("Output Pattern for CSS must end in .css");
   }
-  
+
+  // Default options for HTMLMinifier
+  // https://github.com/kangax/html-minifier#options-quick-reference
+  const defaultHtmlMinifierOptions = {
+    collapseWhitespace: true,
+    removeComments: true,
+    removeEmptyAttributes: true,
+    removeRedundantAttributes: true,
+    removeScriptTypeAttributes: true,
+    removeStyleLinkTypeAttributes: true,
+  };
+
+  const htmlMinifierOptions = args.htmlMinifierOptions === false ? defaultHtmlMinifierOptions : Object.assign({}, defaultHtmlMinifierOptions, args.htmlMinifierOptions)
```

Comment:
If I’m reading this right, isn’t this impossible to disable now? What if someone wants to use `@snowpack/plugin-webpack` but doesn’t want their HTML minified?

Reply:
hawman, you are right, I was thinking wrong, glad you caught that. Sorry, I was doing to many things at once today. Also we really need to add tests to our plugins :D 

I assume this is what we want?

```suggestion
  const htmlMinifierOptions = args.htmlMinifierOptions === false ? false : Object.assign({}, defaultHtmlMinifierOptions, args.htmlMinifierOptions)
```

Reply:
Yup! That looks good to me